### PR TITLE
Ember singleton controllers post

### PR DESCRIPTION
**Pull request by juanazam**

### CODE REVIEW

File: source/_posts/2015-11-03-living-with-singleton-controllers.markdown [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+component when the feature is available.
+
+This is great! but what happens with legacy code with LOTS of behavior implemented
+in controllers? You can either try this approach which might be a bit painful or
+try option 2.
+
+## Option 2 - The ninja way
+
+We are trying to make our controllers reset when we transition from
+route to route, so lets to exactly that:
```
Comment (smartinez87) [inline source/_posts/2015-11-03-living-with-singleton-controllers.markdown]: `to` exactly -> `do` exactly ?

Comment (juanazam) [inline source/_posts/2015-11-03-living-with-singleton-controllers.markdown]: :+1:

**Comment by juanazam** [comment]
It seems we don't have handlebars support on codeblock or at least i haven't beeen able to make it work :sweat_smile:

File: source/_posts/2015-11-03-living-with-singleton-controllers.markdown [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+{% codeblock lang:js %}
+import Ember from 'ember';
+import ResetControllerMixin;
+
+export default Ember.Route.extend(ControllerResetMixin, {
+  // my route code
+})
+{% endcodeblock %}
+
+Awesome, we made it! but wouldn't be great to have this behavior working by
```
Comment (eldano) [inline source/_posts/2015-11-03-living-with-singleton-controllers.markdown]: but wouldn't *it* be great

Comment (juanazam) [inline source/_posts/2015-11-03-living-with-singleton-controllers.markdown]: :+1:

**Comment by juanazam** [comment]
Since the needed handlebars code is very small, I found a workaround.

**Pull request merged**